share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -298,11 +298,15 @@ function(ExternalZephyrProject_Cmake)
   get_cmake_property(sysbuild_cache CACHE_VARIABLES)
   foreach(var_name ${sysbuild_cache})
     if(NOT "${var_name}" MATCHES "^CMAKE_.*")
+      # Perform a dummy read to prevent a false cmake warning about unused
+      # variables being emitted
```

Comment:
could you refer to this CMake issue:
https://gitlab.kitware.com/cmake/cmake/-/issues/24555

Reply:
Added